internal/expr/prepare.go

```diff
@@ -114,7 +119,18 @@ func prepareOutput(ti typeNameToInfo, p *outputPart) ([]fullName, []typeMember,
 		return info, nil
 	}
 
-	addColumns := func(info typeInfo, tag string, column fullName) error {
+	addSQL := func(column string, tm typeMember) error {
```

Comment:
I think the naming here is a bit confusing as `addSQL` is really adding a column to the SQL output and adding it to the PreparedExpr object. Maybe it would be clearer by moving
```
pe.outputs = append(pe.outputs, tm)
```
elsewhere, and changing the name from `addSQL` to `addColumnSQL` or something similar.

Reply:
addColumn has disappeared with the latest updates from main :)